src/pages/ProjectDetailPage.js

```diff
@@ -32,7 +33,12 @@ class ProjectDetailPage extends React.Component {
 
         return (
             <Grid container justify='center' className='ProjectDetailPage'>
-                <Grid item xs={10} md={6}>
+                <Grid 
+                    item 
+                    xs={10} 
+                    md={6} 
+                    className='GridProjectDetailPage'
```

Comment:
When adding class names to DOM elements that aren't a parent container for a React component, we should use a different naming convention so it doesn't cause confusion reading SCSS files that might refer to a capitalize camelcase name 

Suggestion: rename to kebab-case `grid-project-detail-page` or `grid-container`